internal/kafka/command_topic.go

```diff
@@ -38,29 +40,30 @@ func newTopicCommand(cfg *config.Config, prerunner pcmd.PreRunner) *cobra.Comman
 
 	c := &command{clientID: cfg.Version.ClientID}
 
-	if cfg.IsCloudLogin() {
+	if cfg.CheckIsCloudLoginOrOnPremLogin() != nil && !cfg.IsTest {
```

Comment:
Why is `cfg.IsTest` here? This seems like a hacky way to make integration tests pass.

Reply:
many of our integration tests are not actually logged in, then they're routed into this branch. This `isTest` doesn't affect non-test use. But i'll look if i can fix the tests login directly